x-pack/libbeat/management/error.go

```diff
@@ -21,21 +19,13 @@ type ErrorType string
 // ConfigError is the type of error send when an unpack or a blacklist happen.
 var ConfigError = ErrorType("CONFIG")
 
-// ErrorEvent is the event type when an error happen.
-var ErrorEvent = api.EventType("ERROR")
-
 // Error is a config error to be reported to kibana.
 type Error struct {
 	Type ErrorType
 	UUID uuid.UUID
 	Err  error
 }
 
-// EventType returns a ErrorEvent.
-func (e *Error) EventType() api.EventType {
-	return ErrorEvent
-}
-
 // MarshalJSON transform an error into a JSON document.
 func (e *Error) MarshalJSON() ([]byte, error) {
```

Comment:
Are the MarshalJSON/UnmarshalJSON functions used by fleet?

Reply:
@blakerouse @michalpristas Any idea?

Reply:
I do not believe they are used.